streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -2333,74 +2333,6 @@ void runOnce() {
         verify(taskManager);
     }
 
-    @Test
-    @SuppressWarnings("unchecked")
-    public void shouldCatchTaskMigratedExceptionOnOnTaskCorruptedExceptionPath() {
```

Comment:
This test no longer applies, since we don't have a TaskMigratedException to catch anymore